### ISSUE TITLE
Searching for the same person in a database

### ISSUE THREAD

**Issue by aviNrich** [justadudewhohacks/face-api.js#813]
Hi,
I testing the framework for a new project.
My app needs to allow uploading images and index its `face-api.js` output in the database.
After the image was indexed, I want to run a search on the entire database and search for the same person.
Is it possible to search similar/same person over database?

Thanks

**Comment by vladmandic** [comment]
yes, its possible. asked and answered several times, search on the old issues.

**Comment by Alireza17224** [comment]
Hi @vladmandic  can you give the direct answer link for that? 
Best regards

**Comment by Dsalomon-16** [comment]
disculpa como lo has publicado no lo he podido encontrar
Sorry how you posted it, I couldn't find it.
